src/core/network/createNetwork.js

```diff
@@ -50,15 +50,23 @@ export default (config, logger, lifecycle, networkStrategy) => {
      * @param {Object} payload This will be JSON stringified and sent as the post body.
      * @param {boolean} [expectsResponse=true] The endpoint and request mechanism
      * will be determined by whether a response is expected.
+     * @param {boolean} [exitLink=false] The network transport method (sendBeacon vs. fetch)
+     * will be determined by whether this is an exitLink.
      * @returns {Promise} a promise resolved with the response object once the response is
      * completely processed.  If expectsResponse==false, the promise will be resolved
      * with undefined.
      */
-    sendRequest(payload, expectsResponse = true) {
+    sendRequest(payload, expectsResponse = true, exitLink = false) {
       const requestId = uuid();
+      if (expectsResponse && exitLink) {
+        logger.log(
+          `Attempt to get response during exit link.  Assuming no response is expected.`
```

Comment:
I'm not sure if this would make sense or be helpful to a customer. I think they may wonder why we're telling them to "attempt to get response during exit link". Maybe "No response requested due to exit link." and always log it whenever `exitLink: true` regardless of the `expectsResponse` value? Just thinking out loud.